pkgs/leak_tracker_flutter_testing/lib/src/leak_tracking_for_tests.dart

```diff
@@ -0,0 +1,198 @@
+// Copyright (c) 2023, the Dart project authors.  Please see the AUTHORS file
+// for details. All rights reserved. Use of this source code is governed by a
+// BSD-style license that can be found in the LICENSE file.
+
+import 'package:leak_tracker/leak_tracker.dart';
+import 'package:meta/meta.dart';
+
+void _emptyLeakHandler(Leaks leaks) {}
+
+/// Leak tracking settings and helper methods for tests.
+abstract class LeakTrackingForTests {
```

Comment:
Why does this class with only static members exist in the first place? https://dart.dev/effective-dart/design#avoid-defining-a-class-that-contains-only-static-members

Reply:
The rule states 'AVOID', not 'DON'T'.

This rule is violated across Dart and Flutter many times (https://api.dart.dev/stable/3.0.6/dart-developer/Service/Service.html).

I think we need to convince language team to provide a way to group static methods, so that we do not use abstract classes for these purposes.

Reply:
It feels like LeakTrackingForTestsSettings and LeakTrackingForTests should just be one class then? Why do the statics need to be separated out?